src/components/sortableList/Item.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import {Dimensions, Platform, StyleSheet, View} from 'react-native';
+import {PanGestureHandler, PanGestureHandlerGestureEvent} from 'react-native-gesture-handler';
+import Animated, {runOnJS, scrollTo, useAnimatedGestureHandler, useAnimatedReaction, useAnimatedStyle, useSharedValue, withTiming} from 'react-native-reanimated';
+import {getOrder, getPosition} from './Config';
+
+const {height} = Dimensions.get('window');
+const isAndroid = Platform.OS === 'android';
+
+interface ItemProps<T> {
+  item: any;
+  index: number;
+  positions: Animated.SharedValue<number[]>;
+  scrollY: Animated.SharedValue<number>;
+  contentHeight: number;
+  containerHeight?: number;
+  listRef: any;
+  itemHeight: number;
+  onFinish: (list: T[]) => void;
```

Comment:
I suggest having a less vague name here, maybe `onFinishDrag`, `onCompleteDrag`, etc..

Reply:
changed to `onFinishDrag`